Doc/library/zipfile.rst

```diff
@@ -377,6 +377,8 @@ ZipFile Objects
    the new entry. Similarly, *compresslevel* will override the constructor if
    given.
    The archive must be open with mode ``'w'``, ``'x'`` or ``'a'``.
+   The *strict_timestamps* keyword argument allows to zip files older
+   than 1980-01-01 at the cost of setting the timestamp to 1980-01-01.
```

Comment:
Just from the doc, I don't understand if I have to set the parameter to True or False to allow years older than 1980.